spec/create.spec.js

```diff
@@ -35,10 +35,9 @@ var project = path.join(tmpDir, appName);
 
 // Setup and teardown test dirs
 beforeEach(function () {
-    fs.removeSync(project);
-    fs.ensureDirSync(tmpDir);
+    fs.emptyDirSync(tmpDir);
 });
-afterEach(function () {
+afterAll(function () {
```

Comment:
Why this change and why here?

Reply:
Why: Since with `emptyDirSync` we empty `tmpDir` in `beforeEach` so there's no need to delete the tmpDir itself after every test, only at the very end.

Why here: Because this change is kind of enabled by the rich `fs-extra` API :man_shrugging: